vmm/rpc/proto/vmm_rpc.proto

```diff
@@ -231,6 +231,8 @@ service Vmm {
   rpc ShutdownVm(Id) returns (google.protobuf.Empty);
   // RPC to resize a VM
   rpc ResizeVm(ResizeVmRequest) returns (google.protobuf.Empty);
+  // RPC to compute the compose hash, it's helpful for debugging & developing SDK.
+  rpc GetComposeHash(VmConfiguration) returns (AppId);
```

Comment:
This is wierd, the name is ComposeHash but returns app_id, where app_id is bytes20 but compose_hash is bytes32.